a10_neutron_lbaas/a10_config.py

```diff
@@ -28,7 +28,31 @@
 
 class A10Config(object):
 
-    def __init__(self, config_dir=None):
+    def __init__(self, config_dir=None, config=None):
+        if config is not None:
+            self._config = config
+            self._load_config()
+            return
+
+        self._config_dir = self._find_config_dir(config_dir)
+        self._config_path = os.path.join(self._config_dir, "config.py")
+
+        real_sys_path = sys.path
+        sys.path = [self._config_dir]
+        try:
```

Comment:
We're going to need to switch to runpy for this to work with scaling. Nuking sys.path prevents the config.py from importing anything. Doesn't prevent merging this pr.